### PR TITLE
added failure test cases

### PR DESCRIPTION
**Pull request by surabhi-r1**

### CODE REVIEW

File: src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java [hunk 1]
```diff
@@ -46,14 +46,16 @@ void delete() {
 
     }
 
-//    @Test
-//    void deleteError() {
-//        long id = 11;
```
Comment (abhishek-gowda12) [inline src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java:51]: please remove commented code

File: src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java [hunk 2]
```diff
@@ -46,14 +46,16 @@ void delete() {
...
+        Mockito.doThrow(new RuntimeException())
+                .when(taskRepository).deleteById(id);
+
+        Response<?> response = taskService.delete(id);
+        assertEquals(HttpStatus.OK, response.getHttpStatus());
+
+    }
 
     @Test
     void getAll() {
```
Comment (abhishek-gowda12) [inline src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java:61]: add scenario (success || failure)

File: src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java (hunk 1 above)
Comment (surabhi-r1) [reply src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java:51]: completed

File: src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java (hunk 2 above)
Comment (surabhi-r1) [reply src/test/java/com/surabhi/taskapp/service/impl/TaskServiceImplTest.java:61]: done

**Review by abhishek-gowda12 — APPROVED** [review]

**Pull request merged**